README.md

```diff
@@ -1,9 +1,15 @@
-# CAPEv2 service
+[![Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg?sanitize=true)](https://discord.gg/GUAy9wErNu)
+[![](https://img.shields.io/discord/908084610158714900)](https://discord.gg/GUAy9wErNu)
+[![Static Badge](https://img.shields.io/badge/github-assemblyline-blue?logo=github)](https://github.com/CybercentreCanada/assemblyline)
+[![Static Badge](https://img.shields.io/badge/github-assemblyline\_service\_cape-blue?logo=github)](https://github.com/CybercentreCanada/assemblyline-service-cape)
+[![GitHub Issues or Pull Requests by label](https://img.shields.io/github/issues/CybercentreCanada/assemblyline/service-cape)](https://github.com/CybercentreCanada/assemblyline/issues?q=is:issue+is:open+label:service-cape)
+[![License](https://img.shields.io/github/license/CybercentreCanada/assemblyline-service-cape)](./LICENSE)
+# CAPE Service
 
-This repository is an Assemblyline service that submits a file to a CAPEv2 deployment, waits for the submission to
-complete, and then parses the report returned.
+This Assemblyline service submits files to a CAPEv2 deployment and parses the report returned.
 
-**NOTE**: This service **requires extensive additional installation outside of Assemblyline** before being functional. It is **not** preinstalled during a default installation.
+## Service Details
+`NOTE`: This service `requires extensive additional installation outside of Assemblyline` before being functional. It is `not` preinstalled during a default installation.
```

Comment:
Is ** for bold not supported in dockerhub publish content?

Reply:
This is probably a bug on my side for doing a find-and-replace-all, I'll fix that.